.goreleaser.yaml

```diff
@@ -41,11 +46,21 @@ nfpms:
       - rpm
     overrides:
       deb:
-        file_name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Arch }}"
```

Comment:
Debian packages have the `<DebianPackageName>_<VersionNumber>-<DebianRevisionNumber>_<DebianArchitecture>.deb ` naming convention. So let's revert this back to what it was before. Debian arch's are amd64 and arm64:

```
file_name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Arch }}"
```